docs/introduction.md

```diff
@@ -64,7 +64,7 @@ Finally, import the styles.
 import * as styles from './Button.treat.js';
 
 export const Button = ({ text }) => `
-  <button class="${styles.button}">${text}</button>
+  <button className="${styles.button}">${text}</button>
```

Comment:
This is actually not a React component, rather a simple html template string. Check out the [React API](https://seek-oss.github.io/treat/react-api) section for React specific examples.

Reply:
👍